config/secrets/test/private.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
please NEVER push private keys (even test ones), I bet github bots should already have warned you :)

instead, in your readme, you can link to the doc about how to generate keys: https://github.com/oat-sa/bundle-lti1p3/blob/master/doc/quickstart/configuration.md#configure-a-keychain